app/models/gogovan_transport.rb

```diff
@@ -1,4 +1,11 @@
 class GogovanTransport < ActiveRecord::Base
   translates :name
   validates :name_en, presence: true
+
+  def vehical_tag
+    case name_en
+      when '5.5t Truck' then 'mudou'
```

Comment:
Indent `when` as deep as `case`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.